projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -36,15 +60,33 @@ export class ListViewComponent implements OnChanges {
     @Input() displayViewSwitch?: boolean = true;
 
     /**
-     * Click on an item will emit the resource iri
+      * Set to true if multiple resources can be selected for comparison
+      */
+    @Input() withMultipleSelection?: boolean = false;
+
+    /**
+     * Click on checkbox will emit the resource info
      *
-     * @param {EventEmitter<string>} resourceSelected
+     * @param {EventEmitter<FilteredResouces>} resourcesSelected
+     */
+    @Output() multipleResourcesSelected?: EventEmitter<FilteredResouces> = new EventEmitter<FilteredResouces>();
+
+    /**
+     * @deprecated
```

Comment:
Please add a description to the `@deprecated` info e.g. Use `singleResourceSelected` instead

Reply:
done